pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v1/Namespaces.java

```diff
@@ -1161,12 +1168,31 @@ public void getBacklogQuotaMap(
             @ApiResponse(code = 412, message = "Specified backlog quota exceeds retention quota."
                     + " Increase retention quota and retry request")})
     public void setBacklogQuota(
+            @Suspended final AsyncResponse asyncResponse,
             @PathParam("property") String property, @PathParam("cluster") String cluster,
             @PathParam("namespace") String namespace,
             @QueryParam("backlogQuotaType") BacklogQuotaType backlogQuotaType,
             BacklogQuota backlogQuota) {
         validateNamespaceName(property, cluster, namespace);
-        internalSetBacklogQuota(backlogQuotaType, backlogQuota);
+        validateNamespacePolicyOperationAsync(namespaceName, PolicyName.BACKLOG, PolicyOperation.WRITE)
+                .thenCompose(__ -> validatePoliciesReadOnlyAccessAsync())
+                .thenCompose(__ -> internalSetBacklogQuotaAsync(backlogQuotaType, backlogQuota))
+                .thenAccept(__ -> {
+                    asyncResponse.resume(Response.noContent().build());
+                    String jsonQuota;
+                    try {
+                        jsonQuota = ObjectMapperFactory.getThreadLocal().writeValueAsString(backlogQuota);
+                    } catch (JsonProcessingException e) {
+                        jsonQuota = e.getMessage();
+                    }
+                    log.info("[{}] Successfully updated backlog quota map: namespace={}, map={}", clientAppId(),
```

Comment:
Using `backlogQuotaType` should be working fine here.

Reply:
I mean that could you use `backlogQuota` instead of `jsonQuota`? `BacklogQuota` class has `@ToString`.

Reply:
why not? and fixed. thanks!